src/docs/installing-a-coding-environment/2-install-tools-mac.md

```diff
@@ -52,10 +52,17 @@ brew install node
 Then, at the prompt:
 
 ```
-node -v
+node --version
```

Comment:
haha

Reply:
I mean, minor but clearer. Trying to reduce cognitive overhead as much as possible.